client/my-sites/domains/domain-management/edit/navigation/index.jsx

```diff
@@ -251,10 +251,25 @@ class DomainManagementNavigation extends React.Component {
 			return null;
 		}
 
-		const title =
-			domainTypes.MAPPED === domainType
-				? translate( 'Delete domain mapping' )
-				: translate( 'Delete your domain permanently' );
+		let title;
+
+		if ( domainTypes.TRANSFER === domainType ) {
+			const status = domain.transferStatus;
+
+			if ( transferStatus.PENDING_OWNER === status || transferStatus.PENDING_REGISTRY === status ) {
+				return null;
+			}
+
+			if ( status === transferStatus.CANCELLED ) {
+				title = translate( 'Remove transfer from your account' );
+			} else {
+				title = translate( 'Cancel transfer' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Cancel Transfer' )` *ES Score: 12*